docs/schema.md

```diff
@@ -289,6 +289,117 @@ you must also specify a `type`, described in
 Use the following types to indicate how the properties in your schema are
 treated by GCP Marketplace.
 
+### `schemaVersion`
+
+The version of the schema. If you want to
+[support managed updates for your app](https://cloud.google.com/marketplace/docs/partners/kubernetes-solutions/support-managed-updates), the value must be `v2`.
+
+### (Required) `applicationApiVersion`
+
+The version of the Application Custom Resource object. As of November 2019,
+the version is `v1beta1`.
+
+### (Required) `publishedVersion`
+
+The release version for the application, as a string. This must match the
+release tag on all your application images. For example, `'1.0.5'`.
+
+### `publishedVersionMetadata`
+
+Information about the version, shown to users in the GCP Console when they
+view their Kubernetes workloads.
+
+#### `releaseNote`
+
+Information about the release, using the
+[YAML folding style, with a block chomping indicator](https://yaml-multiline.info).
+For example:
+
+```yaml
+releaseNote: >-
+  Bug fixes and performance enhancements.
+```
+
+#### (Optional) `releaseTypes`
+
+A list of release types, which can be one or more of the following:
+
+- `Feature`
+- `BugFix`
+- `Security`. We recommend using `Security` only if the update addresses a
+  critical security issue. In the GCP Console, security updates are displayed
+  more prominently than other types of updates.
+
+#### `recommended`
+
+A boolean that indicates whether the update is recommended, such as for a
+security update. If `true`, users are encouraged to update as soon as possible.
+
+### `managedUpdates`
+
+Use this section to indicate that your deployer supports managed updates. If
+you omit this section, users must manually update their installations.
+
+#### `kalmSupported`
+
+If you want to support managed updates for your application, set this to
+`true`.
+
+### `images`
+
+New for schema v2. Declare the images for your application in this section.
```

Comment:
If we separate v1 and v2 sections, as suggested above, we don't need statements like "New for schema v2".